src/lib/common/JsonHelper.h

```diff
@@ -0,0 +1,68 @@
+#ifndef SRC_LIB_COMMON_JSONHELPER_H
+#define SRC_LIB_COMMON_JSONHELPER_H
+
+/*
+*
+* Copyright 2013 Telefonica Investigacion y Desarrollo, S.A.U
+*
+* This file is part of Orion Context Broker.
+*
+* Orion Context Broker is free software: you can redistribute it and/or
+* modify it under the terms of the GNU Affero General Public License as
+* published by the Free Software Foundation, either version 3 of the
+* License, or (at your option) any later version.
+*
+* Orion Context Broker is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+* General Public License for more details.
+*
+* You should have received a copy of the GNU Affero General Public License
+* along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+*
+* For those usages not covered by this license please contact with
+* iot_support at tid dot es
+*
+* Author: Orion dev team
+*/
+
+#include <sstream>
+#include <string>
+#include <vector>
+
+class JsonHelper
+{
+public:
+  JsonHelper();
+  void addString(const std::string& key, const std::string& value);
+  void addRaw(const std::string& key, const std::string& value);
+  std::string str();
+private:
+ std::ostringstream ss;
+ bool empty;
+};
+
+std::string toJsonString(const std::string& input);
```

Comment:
I know it's not mandatory (it should be), but please use the keyword 'extern' for external declarations of functions in header files. All the rest of the project does it. And ... dont forget the function header. Yes, in header files we use the function headers also